wallet-new/integration/AccountSpecs.hs

```diff
@@ -98,6 +98,67 @@ accountSpecs wRef wc =
 
             map (AccountBalance . accAmount) accsUpdated `shouldBe` balancesPartialUpdated
 
+
+
+        it "redeeming avvv key gives rise to the corresponding increase of balance of wallet'account - mnemonic not used" $ do
+
+            newWallet <- randomWallet CreateWallet
+            Wallet{..} <- createWalletCheck wc newWallet
+
+            --adding new account
+            rAcc <- generate arbitrary :: IO NewAccount
```

Comment:
For now this is OK -- but remember to fix it as part of CBR-408! 😉

Reply:
I will :-)

Reply:
we have a new input - actually doing `generate arbitrary` in the body of property test is not what we want (long term). But I guess this is not place to do the refactorings. @KtorZ will probably organize a call this week to talk about this

Reply:
I already remove all generate/etc from integration test here: https://github.com/input-output-hk/cardano-sl/compare/paweljakubas/CBR-408/improve-integration-tests and hope this one will be merged. And then return to this one and adopt pick instead of generate